types/detect-it/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Thank you for your fast response. I've just removed the file according to your feedback.